holoviews/plotting/comms.py

```diff
@@ -287,8 +287,8 @@ class JupyterCommManager(CommManager):
     JupyterCommManager.prototype.get_client_comm = function(plot_id, comm_id, msg_handler) {
       if (comm_id in window.HoloViews.comms) {
         return HoloViews.comms[comm_id];
-      } else if ((window.Jupyter !== undefined) && (Jupyter.notebook.kernel != null)) {
-        var comm_manager = Jupyter.notebook.kernel.comm_manager;
+      } else if ((window.comm_manager || (window.Jupyter !== undefined) && (Jupyter.notebook.kernel != null))) {
```

Comment:
The bracketing here differs from the one above and I think the logic is therefore not quite correct.

Reply:
Well spotted. Thanks!

Reply:
Fixed.